tests/test_utils.py

```diff
@@ -832,26 +833,13 @@ def test_is_dr_reachable():
     assert len(ground_nsrts) == 8
     atoms = {pred1([cup1, plate1]), pred1([cup1, plate2])}
     ground_nsrts = utils.filter_static_operators(ground_nsrts, atoms)
-    assert utils.is_dr_reachable(ground_nsrts, atoms, {pred1([cup1, plate1])})
-    assert utils.is_dr_reachable(ground_nsrts, atoms, {pred1([cup1, plate2])})
-    assert utils.is_dr_reachable(ground_nsrts, atoms, {pred2([cup1, plate1])})
-    assert utils.is_dr_reachable(ground_nsrts, atoms, {pred2([cup1, plate2])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred3([cup1, plate1])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred3([cup1, plate2])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred1([cup2, plate1])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred1([cup2, plate2])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred2([cup2, plate1])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred2([cup2, plate2])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred3([cup2, plate1])})
-    assert not utils.is_dr_reachable(
-        ground_nsrts, atoms, {pred3([cup2, plate2])})
+    reachable_atoms = utils.get_reachable_atoms(ground_nsrts, atoms)
+    assert {pred1([cup1, plate1]), pred1([cup1, plate2]), pred2([cup1, plate1]),
+            pred2([cup1, plate2])}.issubset(reachable_atoms)
+    assert not reachable_atoms & {pred3([cup1, plate1]), pred3([cup1, plate2]),
```

Comment:
I got confused by order of operations here. Can you make it more clear how the `not` and the set intersection relate?